vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Handling Usage Files
+description: Guidance for handling usage files.
+ms.date: 05/11/2023
+---
+
+# Handling Usage Files
+
+## Overview
+Usage files play an important role in providing instructions, examples, or guidelines for utilizing packages within our repository. To ensure consistency and clarity, we have established the following guidelines for handling usage files:
```

Comment:
Because this is not a maintainer guide, it should be phrased less prescriptively.

Instead of telling users "Just do this" without further context, it should say "You need to provide a file inside the package at XYZ location. This can be done via `configure_file()` ...."